requirements-dev.txt

```diff
@@ -105,7 +105,7 @@ greenlet==2.0.2; platform_machine == 'aarch64' or (platform_machine == 'ppc64le'
 kombu==5.3.2; python_version >= '3.8'
 prompt-toolkit==3.0.39; python_full_version >= '3.7.0'
 pyasn1==0.5.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4, 3.5'
-pydantic==1.10.12; python_version >= '3.7'
+pydantic==1.10.13; python_version >= '3.7'
```

Comment:
@kairoaraujo  why we opt-in to use `pydantic` version `1.X.Y` when there is even `2.4.2` official version out there: https://github.com/pydantic/pydantic/releases?